### DISCUSSION TITLE
Issues when applying education benefits for students in Harbin Institute of Technology(Shenzhen).

### DISCUSSION THREAD

**Discussion by Ryanzexuan** [community/community#48963] [GitHub Education]
Hi,
 I am a student from Harbin Institute of Technology(Shenzhen). The application requires us email has to be followed by - cs.hitsz.edu.cn
or - hitsz.edu.cn or - stmail.hitsz.edu.cn. However as a student from Harbin Institute of Technology(Shenzhen), we do have our own school email account with -stu.hit.edu.cn but not those above. But once I have selected this email , the website automatically fulfill the blank of 'the name of my school' using Harbin Institute of Technology rather than Harbin Institute of Technology(Shenzhen). So there is no way I can pass when the website requires the instant location which now I am currently in Shenzhen campus rather than Harbin campus. Thus the application would always be like 1.The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. 2.You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so. I am quite frustrated by these problems. Please contact me if there is any chance or ideas to help me solve these issues.

**Comment by samus-aran** [comment]
Hi @Ryanswoodhouse 👋 

Schools can have multiple campuses and emails associated with them so although it states Harbin Institute of Technology this would also include Shenzhen. 

When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.

If you are using a VPN, please disable it when you reapply. If you are not using a VPN, try reapplying when connected to the internet from a different network.

Let us know how you get on. 👍
